src/models/BaseLoginModel.js

```diff
@@ -27,6 +27,9 @@ function (Okta, Q) {
         return trans;
       })
       .fail(function(err) {
```

Comment:
Even though this is in the BaseLoginModel, when it's thrown, Q thinks it's unhandled.